src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const short = 2;
+  const abc = 6;
+  const num = 40;
+  const norm = 20;
+  const long = 50;
```

Comment:
according to [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) those should be upperacased, outside of function